source/plugins/ruby/MdmMetricsGenerator.rb

```diff
@@ -37,6 +37,12 @@ class MdmMetricsGenerator
     Constants::MEMORY_WORKING_SET_BYTES => Constants::MDM_NODE_MEMORY_WORKING_SET_PERCENTAGE,
   }
 
+  @@node_metric_name_metric_allocatable_percentage_name_hash = {
+    Constants::CPU_USAGE_MILLI_CORES => Constants::MDM_NODE_CPU_USAGE_ALLOCATABLE_PERCENTAGE,
+    Constants::MEMORY_RSS_BYTES => Constants::MDM_NODE_MEMORY_RSS_ALLOCATABLE_PERCENTAGE,
```

Comment:
you also need to make a change on metrics rp for hardcoding the metric metadata (units & allowed aggs etc...) as first step before you push this change out.

Reply:
ok, will ping you again once I have that change ready

Reply:
metrics rp changes have been made and are rolling out. resolving converstaion.